client/dist/styles.css

```diff
@@ -1,53 +1,111 @@
 #app {
   background-color: lightgray;
   display: flex;
-  margin-left: auto;
-  margin-right: auto;
-}
 
-#imageModalImage #PanAndZoomImage-container{
-  margin: 0;
-  background: black;
-  position: fixed;
-  top: 0%;
-  left: 0%;
-  transition: all .2s ease;
-  /* padding: 100% */
 }
 
-#imageModalBG {
+#ImageModal{
+  z-index: 99 !important;
```

Comment:
Didn't know about !important so this is interesting, if the z-index is that high is !important still necessary?